backend/tests/waterQualityStats.test.js

```
@@ -0,0 +1,123 @@
+      join: jest.fn().mockReturnThis(),
+      where: jest.fn().mockReturnThis(),
+      whereNotNull: jest.fn().mockReturnThis(),
+      clone: jest.fn(() => createQueryBuilder()),
+      count: jest.fn().mockReturnThis(),
+      select: jest.fn().mockReturnThis(),
+      groupBy: jest.fn().mockReturnThis(),
+      avg: jest.fn().mockReturnThis(),
+      distinct: jest.fn().mockReturnThis(),
+      max: jest.fn().mockReturnThis(),
+      first: jest.fn(() => Promise.resolve(mockResult.firstResult)),
+      then: function (resolve, reject) {
```

Biome (2.4.4)

[error] Do not add then to an object.

(lint/suspicious/noThenProperty)